modules/spades/spades.nf

```diff
@@ -14,7 +14,8 @@ process SPADES {
     tuple val(genomeName), path(genomeReads)
 
     output:
-    path("*_contigs.fasta")
+    tuple val(genomeName), path("*_contigs.fasta"), emit: PROKKA
+    path("*_contigs.fasta"), emit: QUAST
```

Comment:
@Mxrcon , this solution doesn't look elegant to forward the same files in two channels using two statements. Perhaps there's a way we can modify `QUAST` process to rely on the same channel type signature?

Reply:
@abhi18av We could modify `quast` input as receiving val(genomeName) won't be a problem to the `quast `process.

Reply:
@Mxrcon , for the time being - I have actually taken a similar approach as you, to avoid some possibly inelegant Channel manipulation.